packages/html/src/dropzone/dropzone.spec.tsx

```diff
@@ -9,14 +9,16 @@ const states = [
 
 export type KendoDropzoneProps = {
     icon?: string;
+    hint?: string | boolean;
     note?: string | boolean;
 };
 
 export type KendoDropzoneState = { [K in (typeof states)[number]]?: boolean };
 
 const defaultProps = {
     icon: "upload",
-    note: "Only JPEG, PNG and SVG files are allowed."
+    hint: "Drag and drop files here to upload",
+    note: "Only JPEG, PNG and SVG files are allowed.",
```

Comment:
Could we make the `note` optional? This way in the normal template we will skip the prop with false value?